plugin/pkg/admission/runtimeclass/OWNERS

```diff
@@ -2,6 +2,4 @@
 
 approvers:
   - tallclair
-reviewers:
-  - egernst
-  - yastij
+reviewers: []
```

Comment:
Is this a desirable thing to have in the file?  Would not having `reviewers` be better?

Reply:
@smarterclayton yes, we can update the automation to do this. will line that as a TODO.